.travis.yml

```diff
@@ -6,162 +6,163 @@ addons:
     update: false
 
 stages:
-  - "EDA Tools - Misc"
-  - "EDA Tools - Bitstream"
-  - "EDA Tools - Simulation"
-  - "EDA Tools - Synthesis"
-  - "EDA Tools - PnR"
-  - "EDA Tools - Formal"
-  - "EDA Tools - Xilinx Vivado metapackages"
+  - "Stage one"
```

Comment:
I know this is a work in progress, but I would highly suggest using more descriptive names than `"Stage one"` and `"Stage two"`....

Reply:
Sure, I'll try to think about something more creative